sdk/postgresql/Azure.ResourceManager.PostgreSql/api/Azure.ResourceManager.PostgreSql.netstandard2.0.cs

```diff
@@ -908,12 +1116,50 @@ public partial class PostgreSqlFlexibleServerEditionCapability
         public static bool operator !=(Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerFailoverMode left, Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerFailoverMode right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class PostgreSqlFlexibleServerFastProvisioningEditionCapability
+    public partial class PostgreSqlFlexibleServerFastProvisioningEditionCapability : Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlBaseCapability
     {
         internal PostgreSqlFlexibleServerFastProvisioningEditionCapability() { }
+        public int? ServerCount { get { throw null; } }
         public string SupportedServerVersions { get { throw null; } }
         public string SupportedSku { get { throw null; } }
-        public long? SupportedStorageGb { get { throw null; } }
+        public int? SupportedStorageGb { get { throw null; } }
```

Comment:
This is reported in apicompat because its type changed from long to int.
Since long and int could be compatible especially here we are downgrading it, we could simply fix this by adding 
public long? SupportedStorageGb into the partial class of this type and regenerate, then the generator could automatically pick this property up as a long instead of int.